scala/scala.bzl

```diff
@@ -452,28 +467,36 @@ env JAVACMD=$0.runfiles/{repo}/{java} $0.runfiles/{repo}/{scala} {jvm_flags} -cl
       runfiles=runfiles)
 
 def _scala_test_impl(ctx):
-  deps = ctx.attr.deps
-  deps += [ctx.attr._scalatest_reporter]
-  jars = _collect_jars(deps)
-  (cjars, rjars) = (jars.compiletime, jars.runtime)
-  cjars += [ctx.file._scalareflect, ctx.file._scalatest, ctx.file._scalaxml]
-  rjars += [ctx.outputs.jar, ctx.file._scalalib, ctx.file._scalareflect, ctx.file._scalatest, ctx.file._scalaxml]
-  rjars += _collect_jars(ctx.attr.runtime_deps).runtime
-  _write_test_launcher(ctx, rjars)
-  return _scala_binary_common(ctx, cjars, rjars)
+    deps = ctx.attr.deps
+    deps += [ctx.attr._scalatest_reporter]
+    jars = _collect_jars(deps)
+    (cjars, rjars) = (jars.compiletime, jars.runtime)
+    cjars += [ctx.file._scalareflect, ctx.file._scalatest, ctx.file._scalaxml]
+    rjars += [
+              ctx.outputs.jar,
+              ctx.file._scalalib,
+              ctx.file._scalareflect,
+              ctx.file._scalatest,
+              ctx.file._scalaxml
+              ]
+    rjars += _collect_jars(ctx.attr.runtime_deps).runtime
+    _write_test_launcher(ctx, rjars)
+    return _scala_binary_common(ctx, cjars, rjars)
 
 _implicit_deps = {
   "_ijar": attr.label(executable=True, default=Label("@bazel_tools//tools/jdk:ijar"), single_file=True, allow_files=True),
   "_scala": attr.label(executable=True, default=Label("@scala//:bin/scala"), single_file=True, allow_files=True),
-  "_scalac": attr.label(executable=True, default=Label("@scala//:bin/scalac"), single_file=True, allow_files=True),
+  "_scalac": attr.label(cfg=HOST_CFG, executable=True, default=Label("//src/java/io/bazel/rulesscala/scalac"), allow_files=True),
```

Comment:
Do you mean we should want to add or remove? I've seen us compiling targets on different actions again, but haven't noticed us just doing the same thing twice over. But i'd believe it..